wicket-core/src/main/java/org/apache/wicket/util/tester/BaseWicketTester.java

```diff
@@ -253,7 +254,7 @@ public BaseWicketTester(final WebApplication application, String servletContextB
 	}
 
 	/**
-	 * Creates a <code>WicketTester</code>.
+	 * Creates a <code>WicketTester</code>. Constructor to keep compatibility.
```

Comment:
There is no need of the new comment. I guess the constructor is a lot and will be used in the future.